eZ/Publish/Core/Persistence/Cache/UrlWildcardHandler.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace eZ\Publish\Core\Persistence\Cache;
+
+use eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler as UrlWildcardHandlerInterface;
+
+class UrlWildcardHandler extends AbstractHandler implements UrlWildcardHandlerInterface
+{
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler::create
+     */
+    public function create($sourceUrl, $destinationUrl, $forward = false)
+    {
+        $this->logger->logCall(
+            __METHOD__,
+            [
+                'sourceUrl' => $sourceUrl,
+                'destinationUrl' => $destinationUrl,
+                'forward' => $forward,
+            ]
+        );
+
+        $urlWildcard = $this->persistenceHandler->urlWildcardHandler()->create($sourceUrl, $destinationUrl, $forward);
+
+        // need to clear lists of UrlWildcards cached due to loadAll()
+        $this->cache->invalidateTags(['ez-urlWildcard-all']);
+
+        return $urlWildcard;
+    }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler::remove
+     */
+    public function remove($id)
+    {
+        $this->logger->logCall(
+            __METHOD__,
+            [
+                'id' => $id,
+            ]
+        );
+
+        $urlWildcard = $this->load($id);
+
+        $this->persistenceHandler->urlWildcardHandler()->remove($urlWildcard->id);
+
+        // need to clear lists of UrlWildcards cached due to loadAll()
+        $this->cache->invalidateTags(['ez-urlWildcard-all']);
+    }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler::load
+     */
+    public function load($id)
+    {
+        $this->logger->logCall(
+            __METHOD__,
+            [
+                'id' => $id,
+            ]
+        );
+
+        $cacheItem = $this->cache->getItem('ez-urlWildcard-id-' . $id);
+
+        if ($cacheItem->isHit()) {
+            return $cacheItem->get();
+        }
+
+        $urlWildcard = $this->persistenceHandler->urlWildcardHandler()->load($id);
+
+        $cacheItem->set($urlWildcard);
+        $cacheItem->tag($this->getCacheTags([$urlWildcard]));
+        $this->cache->save($cacheItem);
+
+        $cacheItemBySource = $this->cache->getItem('ez-urlWildcard-source-' . $this->escapeForCacheKey($urlWildcard->sourceUrl));
```

Comment:
afaik, no need to pre warm the cache for another key here
1. complicates tests
2. fills up cache with cache that might not be used

And last but not least, isth the logic here flawed? `sourceUrl` is typically a pattern right? while url sent to `lookup` is an actual url. So this prewarmed item will never be used .